### DISCUSSION TITLE
Authorization request doesn't contain "request"

### DISCUSSION THREAD

**Discussion by lucamontano98** [OpenIDC/mod_auth_openidc#1127] [General]
Hello,
after set up configuration file with all data indicated in documentation I don't receive "request" claim when RP call my Provider. How can I have that claim in request? Thanks.

Luca

**Comment by zandbelt** [comment]
hard to tell without the config file, the log file and the module version info, but you should configure `OIDCPublicKeyFiles`, `OIDCPrivateKeyFiles` and `OIDCRequestObject` to make mod_auth_openidc generate an authorization request with a request object as defined in https://openid.net/specs/openid-connect-core-1_0.html#RequestParameter; if that does't work for you, the server debug log would probably tell you why

**Comment by lucamontano98** [comment]
ok, can you tell me where can I find documentation about how to use OIDCPublicKeyFiles, OIDCPrivateKeyFiles and OIDCRequestObject and how to generate client metadata? Thanks.

Luca

**Reply by zandbelt** [reply]
https://github.com/OpenIDC/mod_auth_openidc/blob/master/auth_openidc.conf
